Update Configure [1.0]:

aikuci opened the pull request:
- Add Cypress Config
includes cypress.json, jsconfig.json
- Update package
- Update gitignore

#2

ilham76c reviewed (approved):
Okey, penambahan konfigurasi sudah di cek

ilham76c commented:
konfigurasi saya merge ke branch develop #2

achsan committed:
Update Configure [1.0]:
- Add Cypress Config
includes cypress.json, jsconfig.json
- Update package
- Update gitignore
